CLAUDE.md

```
@@ -0,0 +1,124 @@
+./gradlew test                                         # 전체 테스트
+./gradlew test --tests <fully.qualified.TestClass>     # 단일 클래스 테스트
+./gradlew bootRun                                      # 애플리케이션 실행 (local 프로파일)
+```
+
+로컬 인프라(PostgreSQL, Redis, MinIO)가 없으면 테스트가 실패할 수 있다. 인프라 부재로 인한 실패와 코드 문제로 인한 실패를 명확히 구분해서 보고한다.
+
+## Architecture Overview
+
+### Package Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)